tests/test_model.py

```diff
@@ -55,13 +55,17 @@
 from tests.core.conftest import DEFAULT_DOMAIN_PATH_WITH_MAPPING, DEFAULT_STACK_CONFIG
 
 
-def test_get_latest_model(trained_rasa_model: str):
-    path_of_latest = os.path.join(os.path.dirname(trained_rasa_model), "latest.tar.gz")
-    shutil.copy(trained_rasa_model, path_of_latest)
+def test_get_latest_model(tmpdir: Path):
```

Comment:
`tmpdir` is not of type `Path` but of type `LocalPath` which is slightly different. I suggest to use always `tmp_path`
```suggestion
def test_get_latest_model(tmp_path: Path):
```

Reply:
That's been amended & also the `test_test_core_comparison_after_train` was changed too 🥳  Let me know your thoughts.